chore: Rename references to Storage Manager in docs to File Uploader (#5697)
aws-amplify/amplify-ui: docs/src/pages/[platform]/connected-components/storage/fileuploader/index.page.mdx

---

> title: File Uploader
> description: The File Uploader lets your users upload and manage files to the cloud.

Reviewer: ```suggestion
title: FileUploader
description: The FileUploader lets your users upload and manage files in the cloud.
```

Author: 👍
